src/erlfmt_cli.erl

```diff
@@ -30,10 +31,10 @@ opts() ->
         {write, $w, "write", undefined, "modify formatted files in place"},
         {out, $o, "out", binary, "output directory"},
         {verbose, undefined, "verbose", undefined, "include debug output"},
-        {check, $c, "check", undefined,
+        {check, undefined, "check", undefined,
```

Comment:
Why this change?

Reply:
I don't know, it just seems more symmetric with --verbose

Reply:
`--verbose` doesn't use `-v` becuase it's already used for `--version`. We could use `-V` for alias to `--verbose`

Reply:
In general, I think it is okay to leave out the short hands for everything that isn't `-h`.
But I am okay with add the `-c` back, but `-V` seems weird.

Reply:
Added back -c